home.html

```diff
@@ -35,16 +35,59 @@
           </ul>
         </div>
         <i class="fa fa-bars" onclick="showMenu()"></i>
-        <!-- Building the burger icon below-->
-        <!-- <i class="fa fa-bars"></i> -->
       </nav>
-      <div class="text-box">
+      <!-- Slideshow container -->
+      <div class="slideshow-container">
+        <!-- Full-width images with number and caption text -->
+        <div class="mySlides fade">
```

Comment:
@AmaniLouendriz not a requirement, but a lot of times these days, people use dashes for their class names, so in this case it would be `<div class="my-slides fade">`.